### ISSUE TITLE
Inconsistent GUI interactions between photos and search

### ISSUE THREAD

**Issue by justin13888** [immich-app/immich#9573]
### The bug

On the Photos page (/photos), I could do things like select and use delete shortcut. But I'm unable to do the same interactions on the search page (/search)

### The OS that Immich Server is running on

Ubuntu 22.04

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

v1.105.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
One example is when you select one asset and shift click another, it will be a ranged selection on the photos page but just two individual selections on the search page
```

**Comment by jasonhun** [comment]
Agreed, shift-select in search results would be really nice when trying to archive big sets of photos like photos of receipts and other things I don't really care to see unless I absolutely, but unlikely need to.  Clicking thousands of them individually is really time consuming.

**Comment by TomPohys** [comment]
+1

I would like to use `shift` feature for tagging multiple photos that are not in any album (in /search) for cleaning up my photos.
